generate_long_video.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+from utils import settings
+from video_generator import VideoGenerator
+import sys
+
+abs_directory = Path().absolute()
+config = settings.check_toml(f"{abs_directory}/utils/.config.template.toml", f"{abs_directory}/config.toml")
+
+num_of_vids_to_be_conc = input("How many videos would you like to be concatenated into a single video? ")
```

Comment:
Maybe get this from the config? It would be nicer to automate

Reply:
Hi.

I implemented this so that the user can choose essentially how long the video should be without having to update a file over and over again. If you would like it then I can make the change